libraries/knapsack_pro-ruby/guide.md

```diff
@@ -980,7 +980,104 @@ If you are going to relay on rspec to autobalance build when cucumber tests were
   </p>
 
-  <script src="https://gist.github.com/ArturT/a35284f34c2dc0b61a0ad2b4dd4bacae.js"></script>
+{% highlight yaml %}
+# .github/workflows/main.yaml
+name: Main
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    # If you need DB like PostgreSQL, Redis then define service below.
+    # https://github.com/actions/example-services/tree/master/.github/workflows
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: ""
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        # tmpfs makes DB faster by using RAM
+        options: >-
+          --mount type=tmpfs,destination=/var/lib/postgresql/data
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+        options: --entrypoint redis-server
+
+    # https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+    strategy:
+      fail-fast: false
+      matrix:
+        # [n] - where the n is a number of parallel jobs you want to run your tests on.
+        # Use a higher number if you have slow tests to split them between more parallel jobs.
+        # Remember to update the value of the `ci_node_index` below to (0..n-1).
+        ci_node_total: [2]
+        # Indexes for parallel jobs (starting from zero).
+        # E.g. use [0, 1] for 2 parallel jobs, [0, 1, 2] for 3 parallel jobs, etc.
+        ci_node_index: [0, 1]
+
+    env:
+      RAILS_ENV: test
+      GEMFILE_RUBY_VERSION: 2.7.2
+      PGHOST: localhost
+      PGUSER: postgres
+      # Rails verifies the time zone in DB is the same as the time zone of the Rails app
+      TZ: "Europe/Warsaw"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          # Not needed with a .ruby-version file
+          ruby-version: 2.7
+          # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+
+      - name: Create DB
+        run: |
+          bin/rails db:prepare
+      - name: Run tests
+        env:
+          KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC }}
+          KNAPSACK_PRO_TEST_SUITE_TOKEN_CUCUMBER: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_CUCUMBER }}
+          KNAPSACK_PRO_TEST_SUITE_TOKEN_MINITEST: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_MINITEST }}
+          KNAPSACK_PRO_TEST_SUITE_TEST_UNIT: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_TEST_UNIT }}
+          KNAPSACK_PRO_TEST_SUITE_TOKEN_SPINACH: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_SPINACH }}
+          KNAPSACK_PRO_CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
```

Comment:
Please note that when you use jekyll to render code it can't properly render this line:

```yml
KNAPSACK_PRO_CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
```

due to this syntax: `${{ something }}`. It won't be displayed.

The only way to render this is to use raw format.

```
{% highlight yml %}
{% raw %}
...
KNAPSACK_PRO_CI_NODE_TOTAL: ${{ matrix.ci_node_total }} # now this line will be render properly
...
{% endraw %}
{% endhighlight %}
```

The downside of using the raw format is no code highlighting. There are no colors and the code is less readable for the user.

That's why I was using github gist to have code colors highlighting and to render properly line with chars `${{ something }}`.

Maybe you will find a solution to how to render colors in jekyll and make chars visible `${{ something }}`.

Reply:
1. Thank you for noticing this.
2. The
```
{% highlight yml %}
{% raw %}
```
construct solves the issue (by disabling [Liquid](https://jekyllrb.com/docs/liquid/)  _and_ keeps the color highlighting.). I've updated the PR

Reply:
BTW, I considered just creating the gists under the organization account, but unfortunately this [seems impossible to do](https://stackoverflow.com/questions/20647454/how-can-you-create-github-gists-that-live-under-an-organization-account-instead)